src/recipes/dawet-rice-flours.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Dawet is a special drink from Javanese Island, its made from rice flour and cornstarch. It serve with brown sugar sauce.",
+    "description": "Es Dawet",
```

Comment:
One more suggestion: The drink name should just be the name. The rest can go into the description.